tests/Integration/TracerTest.php

```diff
@@ -21,6 +22,7 @@ protected function setUp()
     protected function tearUp()
     {
         \putenv('DD_TRACE_GLOBAL_TAGS');
+        \putenv('DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED');
         parent::tearDown();
     }
 
```

Comment:
Yes, `tearUp` should be `tearDown` for sure. Good catch. :)

Reply:
Good catch 😄! The test is not failing because in this case I was doing a `putenv` at the begin of relevant tests that do override that value. Nonetheless, it is a good habit to left the env clean after every test.